pkg/csi_driver/node.go

```diff
@@ -67,7 +68,7 @@ func (s *nodeServer) NodePublishVolume(ctx context.Context, req *csi.NodePublish
 
 	// Check if mount already exists
 	mounted, err := s.isDirMounted(targetPath)
-	if err != nil {
+	if err != nil && !os.IsNotExist(err) {
```

Comment:
maybe have `isDirMounted` return `false, nil` if the directory doesn't exist

Reply:
If `isDirMounted` returns `false, nil` if the directory doesn't exist there is code in [`unmountPath`](https://github.com/almos98/gcp-filestore-csi-driver/blob/83a40d3799e620f682bb3dca054f139359877f80/pkg/csi_driver/node.go#L179) that might not be reached. What should I do about that code?

Reply:
I think the unmountPath logic would work fine.  If the directory doesn't exist, `unmountPath` returns success. We would just lose the log message, which maybe could be added into `isDirMounted` if we wanted to keep it.

Reply:
Currently don't keep the log message, let me know if you'd prefer to keep it.